morph/cli.py

```diff
@@ -78,6 +80,23 @@ def db_path(db_name):
     return os.path.join(profile_path(), 'dbs', db_name + '.db')
 
 
+def load_db(db_name, allow_missing=False):
+    path = db_path(db_name)
+    if not allow_missing and not os.access(path, os.R_OK):
+        die('can\'t read db file: %s' % (path,))
+    return MorphDb(path, ignoreErrors=allow_missing)
+
+
+def parse_morpheme(morpheme_str):
+    fields = morpheme_str.split('\t')
+    if len(fields) != 4:
+        die('bad morpheme (should be 4 tab-separated fields, MorphMan-style): %s'
+            % (morpheme_str,))
+    # Only 4 fields show up in equality, hash, or show, but the constructor
+    # takes an extra one.  That's the second argument; duplicate the first.
```

Comment:
(Just to be sure I understand you: I believe this comment doesn't lead to anything that should change in this PR.)